packer/run-nginx.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo nginx
```

Comment:
I think you can scrap this file and just run `sudo nginx` from the user-data script directly.